docs/guides/advanced/README.md

```diff
@@ -6,7 +6,7 @@ tags: [""]
 
 ## Software
  - [Home Assistant](/guides/advanced/home_assistant.html) Sample automations to work with StateMonitor

```

Comment:
`[Home Assistant](/guides/advanced/home_assistant.html) sample automations to work with [State Monitor](/docs/firmware/state-monitor-esp32.html)`